### DISCUSSION TITLE
Deadline

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#3] [Announcements]
**Hạn chót TLCH là 9h sáng mai nha**
Ai có việc bận, hong kịp giờ thì báo liền nha!

**Comment by letu25082006** [comment]
mỗi người trl all hay share ra bà?

**Reply by baonhu1304** [reply]
có share sẵn trên tên cụa mỗi bạn gòi é

**Comment by baonhu1304** [comment]
Thôi mình để chiều chiều cho thoải mái thời gian nha mn 😊😋

**Reply by Lenga10A2** [reply]
Ok nhưng mà ủa là làm cái gì v bà

**Reply by baonhu1304** [reply]
trên team, trong kênh riêng của môn Tin có á nha

**Comment by thaopham2501** [comment]
bao giờ mới hết bị dí deadline :')
